e2e_tests/tests/all_providers/normal.rs

```diff
@@ -116,6 +116,7 @@ fn list_opcodes() {
 
     let mut crypto_providers_hsm = HashSet::from_iter(common_opcodes);
     let _ = crypto_providers_hsm.insert(Opcode::CanDoCrypto);
+    let _ = crypto_providers_hsm.insert(Opcode::PsaGenerateRandom);
```

Comment:
All providers support genRandom now. So, it can be moved into `common_opcodes`. And `CanDoCrypto` as well.

Reply:
Okay. Will include them in common opcodes.